test/api_integration/apis/general/csp.js

```diff
@@ -34,16 +34,14 @@ export default function ({ getService }) {
         return [key, parts];
       }));
 
-      // ensure script-src uses a nonce, and remove it so we can .eql everything else
+      // ensure script-src uses self
```

Comment:
We can just remove lines 37-40. They were originally added as a work-around to the nonce being dynamic which prevented us from using just `.to.eql([]`